eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/persistence/model/Reservation.java

```diff
@@ -37,4 +42,7 @@
 
   @ManyToOne
   private MeetingRoom meetingRoom;
+
+  @ManyToMany
```

Comment:
Biztos?

Reply:
Kicseréltem @OneToMany-re.

Reply:
Mapped by vagy cascade lett beállítva?

Reply:
Leteszteltem @OneToMany-vel nem működik. @ManyToMany-vel létrehoz egy kapcsolótáblát és akkor küldi az emailt rendesen. Annyi, hogy resource-ban amikor bejön a Reservation, akkor leellenőrözöm az összes résztvevőt, hogy kerüljenek be a participant táblába ha még nincsenek benne és így már a kapcsolótáblában szépen egymáshoz rendeli őket. A másik csapat is hasonlóan csinálta.